### PR TITLE
Improve sentence in HA section

### PR DESCRIPTION
**Pull request by rgugliel-da**
* the domain does not return 'sequenced endpoints'
* the domain is not the ledger

### CODE REVIEW

File: docs/2.7.9/docs/deploy-daml/infrastructure-architecture/high-availability/basic-ha/domain.rst [hunk 1]
```diff
@@ -20,7 +20,7 @@ Sequencer
 
 Sequencer HA depends on the chosen implementation. For example, when using a ledger such as `Hyperledger Fabric <../../../../canton/usermanual/domains/fabric.html>`_, HA is already set up. Multiple sequencer nodes must be deployed. 
 
-The domain returns multiple sequenced endpoints, any of which can be used to interact with the underlying ledger.
+The domain expose multiple sequencer endpoints, any of which can be used to submit sequencing requests or read events.
```
Comment (oliverse-da) [inline docs/2.7.9/docs/deploy-daml/infrastructure-architecture/high-availability/basic-ha/domain.rst:23]: Looks good, except here and in the other two changes below.

```suggestion
The domain exposes multiple sequencer endpoints, any of which can be used to submit sequencing requests or read events.
```

**Review by oliverse-da — APPROVED** [review]
Looks good - minor typo below

**Review by carrielaben-da — APPROVED** [review]
All good once the previously noted change is made.

File: docs/2.7.9/docs/deploy-daml/infrastructure-architecture/high-availability/basic-ha/domain.rst (hunk 1 above)
Comment (rgugliel-da) [reply docs/2.7.9/docs/deploy-daml/infrastructure-architecture/high-availability/basic-ha/domain.rst]: :see_no_evil:

**Review by yves-da — DISMISSED** [review]
LGTM.

**Pull request merged**